### PR TITLE
EIP-2: updating docs to copy correct _application.properties file

### PR DESCRIPTION
**Pull request by Ruhanga**
Ticket: https://issues.openmrs.org/browse/EIP-2

### CODE REVIEW

File: distribution/springboot_setup/README.md [hunk 1]
```diff
@@ -50,10 +50,10 @@ By default, the exchange of data between the sender and the receiver is done wit
 
 Copy and rename the _application.properties_ files:
 ```
-cp distribution/springboot_setup/sender/application.properties app/src/main/resources/application-sender.properties
+cp distribution/springboot_setup/sender/_application.properties app/src/main/resources/application-sender.properties
```
Comment (wluyima) [inline distribution/springboot_setup/README.md:53]: These files were rename already in master

File: distribution/springboot_setup/README.md [hunk 2]
```diff
@@ -50,10 +50,10 @@ By default, the exchange of data between the sender and the receiver is done wit
...
 ```
 ```
-cp distribution/springboot_setup/receiver/application.properties app/src/main/resources/application-receiver.properties
+cp distribution/springboot_setup/receiver/_application.properties app/src/main/resources/application-receiver.properties
```
Comment (wluyima) [inline distribution/springboot_setup/README.md:56]: These files were rename already in master

File: distribution/springboot_setup/README.md (hunk 1 above)
Comment (Ruhanga) [reply distribution/springboot_setup/README.md:53]: Yes, unfortunately, this was not reflecting in the README.

File: distribution/springboot_setup/README.md (hunk 2 above)
Comment (Ruhanga) [reply distribution/springboot_setup/README.md:56]: Yes, unfortunately, this was not reflecting in the README.

File: distribution/springboot_setup/README.md (hunk 1 above)
Comment (wluyima) [reply distribution/springboot_setup/README.md:53]: i see

**Review by Ruhanga — COMMENTED** [review]
Hi @wluyima, could you please merge this in? Thanks.

**Comment by rbuisson** [comment]
@wluyima , could you merge this ticket?

**Comment by rbuisson** [comment]
From today's meeting, we can merge this PR.

**Pull request merged**